src/App.js

```diff
@@ -21,7 +21,7 @@ export default class App extends Component {
       error:'',
     };
   }
- 
+
   handleButtonClick = async () => {
     try {
       const API_URL = `https://us1.locationiq.com/v1/search.php?key=${process.env.REACT_APP_CITY_KEY}&q=${this.state.exploreQuery}&format=json`;
```

Comment:
Possibly add some logic for clearing the error message/state if the user proceeds to enter another accepted value into the form.

Reply:
The entire function was supposed to be cropped for the comment above.